src/OmniSharp.Roslyn.CSharp/Services/Completion/CompletionListBuilder_Async.cs

```diff
@@ -46,8 +46,13 @@ internal static partial class CompletionListBuilder
                 bool hasAfterInsertStep = false;
                 if (completion.IsComplexTextEdit)
                 {
-                    // The completion is somehow expensive. Currently, this one of two categories: import completion, or override/partial
-                    // completion.
+                    // To-do: Add support for snippet items: https://github.com/OmniSharp/omnisharp-roslyn/issues/2485
+                    if (completion.GetProviderName() == SnippetCompletionProvider)
```

Comment:
Including snippets seems to be causing duplicate items in the completion list, so skipping for now.